c/cmake_modules/version.rc.in

```diff
@@ -0,0 +1,27 @@
+#include <windows.h>
```

Comment:
Could you add our license header?

Reply:
It's for our source code not generated version information.
See also: https://www.apache.org/legal/src-headers.html#headers

Reply:
Apologies, I misunderstood your original comment. Licence comment added to file.